<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorOutputWidget</class>
 <widget class="QWidget" name="InspectorOutputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>317</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item row="4" column="0">
    <widget class="QLabel" name="labelDelayField">
     <property name="text">
      <string>Delay</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelChannelField">
     <property name="text">
      <string>Channel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelVideolayerField">
     <property name="text">
      <string>Video layer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="spinBoxVideolayer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="checkBoxAllowRemoteTriggering">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Allow Remote Triggering</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelDeviceName">
     <property name="text">
      <string>Server</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayoutDevice">
     <item>
      <widget class="QComboBox" name="comboBoxDevice">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelDurationField">
     <property name="text">
      <string>Duration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayoutDuration">
     <item>
      <widget class="QSpinBox" name="spinBoxDuration">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDurationMillisecond">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayoutDelay">
     <item>
      <widget class="QSpinBox" name="spinBoxDelay">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDelayMillisecond">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="lineEditRemoteTriggerId">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelTarget">
     <property name="text">
      <string>Target</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="TargetComboBaseWidget" name="comboBoxTarget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spinBoxChannel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="checkBoxAllowGpi">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Allow GPI</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelRemoteTriggerIdField">
     <property name="text">
      <string>UID</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TargetComboBaseWidget</class>
   <extends>QComboBox</extends>
   <header>Inspector/TargetComboBaseWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxDevice</tabstop>
  <tabstop>comboBoxTarget</tabstop>
  <tabstop>spinBoxChannel</tabstop>
  <tabstop>spinBoxVideolayer</tabstop>
  <tabstop>spinBoxDelay</tabstop>
  <tabstop>spinBoxDuration</tabstop>
  <tabstop>checkBoxAllowGpi</tabstop>
  <tabstop>checkBoxAllowRemoteTriggering</tabstop>
  <tabstop>lineEditRemoteTriggerId</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxChannel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>channelChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxVideolayer</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>videolayerChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxDelay</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>delayChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAllowGpi</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>allowGpiChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxDevice</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>deviceNameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxTarget</sender>
   <signal>editTextChanged(QString)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>targetChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAllowRemoteTriggering</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>allowRemoteTriggeringChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditRemoteTriggerId</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>remoteTriggerIdChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxDuration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorOutputWidget</receiver>
   <slot>durationChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>channelChanged(int)</slot>
  <slot>videolayerChanged(int)</slot>
  <slot>delayChanged(int)</slot>
  <slot>allowGpiChanged(int)</slot>
  <slot>deviceNameChanged(QString)</slot>
  <slot>targetChanged(QString)</slot>
  <slot>allowRemoteTriggeringChanged(int)</slot>
  <slot>remoteTriggerIdChanged(QString)</slot>
  <slot>durationChanged(int)</slot>
 </slots>
</ui>
